model/storage-private/src/main/java/org/keycloak/storage/datastore/DefaultExportImportManager.java

```diff
@@ -813,11 +815,15 @@ public void updateRealm(RealmRepresentation rep, RealmModel realm) {
             realm.setPasswordPolicy(PasswordPolicy.parse(session, rep.getPasswordPolicy()));
         if (rep.getOtpPolicyType() != null) realm.setOTPPolicy(toPolicy(rep));
 
-        WebAuthnPolicy webAuthnPolicy = getWebAuthnPolicyTwoFactor(rep);
-        realm.setWebAuthnPolicy(webAuthnPolicy);
+        if (rep.getWebAuthnPolicyRpEntityName() != null && !rep.getWebAuthnPolicyRpEntityName().isEmpty()) {
```

Comment:
This is why I'm afraid of fixing this globally. I guess you added this because for this one it behaves differently, right?

Reply:
This is slightly orthogonal to the attribute fix. The WebAuthn policy is regenerated regardless of whether or not the attributes have changed. If the new RealmRepresentation has no WebAuthn attributes then the current WebAuthn policy is replaced by the defaults.